src/main/java/com/blackducksoftware/integration/hub/jenkins/PostBuildHubScan.java

```diff
@@ -199,6 +202,15 @@ public boolean perform(final AbstractBuild<?, ?> build, final Launcher launcher,
         return true;
     }
 
+    private boolean isFailureConditionsConfigured(final AbstractBuild<?, ?> build) {
+        for (final Publisher publisher : ((FreeStyleProject) build.getParent()).getPublishersList()) {
+            if (publisher instanceof HubFailureConditionStep) {
```

Comment:
That is only feasible now because we have a single checkbox.  It is safer to check if HubFailureConditionStep is part of the configuration, if it is then its going to do the wait.